docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -714,7 +716,7 @@ If you clone a fork, add the original repository as a remote ``upstream``::
     cd ~/dev/ansible/collections/ansible_collections/community/general
     git remote add upstream git@github.com:ansible-collections/community.general.git
 
-Now you can use this checkout of ``community.general`` in playbooks and roles with whichever version of Ansible you have installed locally, including a local checkout of the ``devel`` branch.
+Now you can use this checkout of ``community.general`` in playbooks and roles with whichever version of Ansible you have installed locally, including a local checkout of the ``main`` branch.
```

Comment:
Here `devel` was referring to an Ansible checkout, not to `community.general`. I guess this needs to be formulated in a better way...

Reply:
gah.. sorry about that. patched it in https://github.com/ansible/ansible/pull/71342